src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -286,7 +217,7 @@ export class StudyViewPageStore {
     private _clinicalDataEqualityFilterSet = observable.shallowMap<ClinicalDataEqualityFilter>();
     private _clinicalDataIntervalFilterSet = observable.shallowMap<ClinicalDataIntervalFilter>();
 
-    @observable private logScaleState = observable.map<boolean>();
+    @observable private _clinicalDataBinFilterSet = observable.map<ClinicalDataBinFilter>();
```

Comment:
@zhx828 Please make sure that toggle log scale checkbox is still functional for bar charts

Reply:
@onursumer updated, it should now work properly